sailfishusermanagerinterface.h

```diff
@@ -30,9 +30,12 @@
 #define SailfishUserManagerErrorUserModifyFailed "org.sailfishos.usermanager.Error.UserModifyFailed"
 #define SailfishUserManagerErrorUserRemoveFailed "org.sailfishos.usermanager.Error.UserRemoveFailed"
 #define SailfishUserManagerErrorGetUidFailed "org.sailfishos.usermanager.Error.GetUidFailed"
+#define SailfishUserManagerErrorGetUuidFailed "org.sailfishos.usermanager.Error.GetUuidFailed"
```

Comment:
For consistency, please name this similarly to others, i.e. UUID instead of Uuid.

Reply:
Ok, or may be Uuid like qt class QUuid?

Reply:
That'd be okay too but then change all instances of UUID to Uuid here and also on D-Bus interfaces.

Reply:
Ok done